### PR TITLE
[FIX][web_m2x_options] Fix Qweb templates.

### PR DESCRIPTION
**Pull request by yajo**
They were trying to replace non-existing elements, and this was being logged to console:

    Can't find "a.oe_m2o_cm_button" when extending template FieldMany2One
    Can't find "span.badge" when extending template FieldMany2ManyTag

@Tecnativa

### CODE REVIEW

File: web_m2x_options/__openerp__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 {
     "name": 'web_m2x_options',
-    "version": "9.0.1.0.0",
+    "version": "9.0.1.0.1",
```
Comment (pedrobaeza) [inline web_m2x_options/__openerp__.py]: Don't you need to update?

Comment (yajo) [reply web_m2x_options/__openerp__.py]: Update what? The addon?

Comment (pedrobaeza) [reply web_m2x_options/__openerp__.py]: Yes, I mean that if it's the case, you need to put 9.0.1.1.0

**Comment by yajo** [comment]
It turns out that making the addon "work" makes `many2many_tags` widget fail. Gonna fix.

**Comment by yajo** [comment]
This should be OK now. Tested it works in production.

**Review by pedrobaeza — APPROVED** [review]

**Review by njeudy — APPROVED** [review]
- test and code review

**Review by rafaelbn — APPROVED** [review]
thanks

**Pull request merged**